src/index/repositories.props

```diff
@@ -13,6 +13,15 @@
         "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat" &amp; build -SkipTests
       </PrepareCommand>
     </Repository>
+    <Repository Include="corefxlab">
+      <Url>https://github.com/dotnet/corefxlab</Url>
+      <Projects>
+        src/*.csproj;
```

Comment:
This will need to be src/**/*.csproj. As its written it will not select any projects